src/FeeRouter/RewardRouter.sol

```diff
@@ -0,0 +1,71 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.16;
+
+interface IInbox {
+    function createRetryableTicket(
+        address to,
+        uint256 l2CallValue,
+        uint256 maxSubmissionCost,
+        address excessFeeRefundAddress,
+        address callValueRefundAddress,
+        uint256 gasLimit,
+        uint256 maxFeePerGas,
+        bytes calldata data
+    ) external payable returns (uint256);
+}
+
+contract RewardRouter {
```

Comment:
Likewise maybe ChildChainReceiver is a better name

Reply:
Same as the other comment, this should be ParentChainReceiver instead